string frame_id
string id
geometry_msgs/Point size
geometry_msgs/Point position
geometry_msgs/Quaternion orientation
---